examples/proc_raw.py

```diff
@@ -0,0 +1,74 @@
+"""
+Show data with postproc
+====================
+`py proc_raw_data.py NODENAME FILENAME EXP_TYPE`
+
+Fourier transforms (and any needed data corrections for older data) are performed according to the `postproc_type` attribute of the data node.
+This script plots the result, as well as signal that's averaged along the `nScans` dimension.
+
+Tested with:
+
+``py proc_raw.py echo_6 240620_200uM_TEMPOL_pm_echo.h5 ODNP_NMR_comp/Echoes``
```

Comment:
This doesn't work for me I get "I caught and error but am plotting anyways and nothing shows up in the resulting window

Reply:
Then debug

Reply:
note that this does work OK on my laptop (remotely) and on the NMR computer.

Reply:
needed to pull pyspecdata updates